lib/utils.pm

```diff
@@ -258,6 +259,23 @@ sub get_netboot_mirror {
     return get_var('MIRROR_' . uc($m_protocol));
 }
 
+sub zypper_call {
+    my ($command, $order) = @_;
+    $order = $order // 1;
+    script_run("zypper -n $command; echo worked-patch-\$?- > /dev/$serialdev", 0);
+    my $ret = wait_serial qr/worked-patch-\d+-/, 2000;
+
+    if ($ret =~ /worked-patch-(\d+)/) {
+        if ($order == 2) {
```

Comment:
Can you use something meaningful for `$order`? Like named parameter? Arbitrary 1 or 2 doesn't hint what it does.  

Why do you need to explicitly check for zypper restart anyway? You can pass exit code as a return value for the `zypper_call` and check that in the test code itself.